### ISSUE TITLE
Importing a gif with a single frame uses default speed, not the gif's frame speed

### ISSUE THREAD

**Issue by BackYard321** [YoYoGames/GameMaker-Bugs#2652]
### Description

When creating a sprite: if you import a gif with multiple frames, that sprite's speed will be set according to the overall speed of the gif. However, if it has a single frame, it will use the default sprite speed instead. If you have code that triggers on animation end of this sprite, this can lead to unexpected behaviour, as the speed that single-frame sprite is using will differ from that of the original gif.

### Steps To Reproduce

1. Start GameMaker and create a new project
2. Import a gif that uses a single frame
3. See the issue
[Contact us.zip](https://github.com/YoYoGames/GameMaker-Bugs/files/13844100/Contact.us.zip)

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2023.11.1 (Monthly)

### Which operating system(s) are you seeing the problem on?

Windows 11

### Are you running GameMaker from inside your Steam library?

None

### Contact Us Package Attached?

- [X] I have attached my Contact Us Package

### Sample Project Added?

- [ ] I have included a small sample project

**Comment by stuckie** [comment]
We would need an example gif to verify this, in case the original doesn't have the data we are requiring.

**Comment by BackYard321** [comment]
> We would need an example gif to verify this, in case the original doesn't have the data we are requiring.

Of course! Here are two gifs to test imports with. Both gifs' "duration" is set to 1000 milliseconds each frame in aseprite. The 2-frame gif is converted to 2fps when imported into GM, but the 1-frame gif is set to the default (in my case, 3 fps. i believe the default is 30 normally?)

![test gif](https://github.com/YoYoGames/GameMaker-Bugs/assets/40671879/19e56779-94de-428f-8cad-7ff48f0e7577)
![test gif 2](https://github.com/YoYoGames/GameMaker-Bugs/assets/40671879/47052e2a-3874-4c06-b8fe-dd30bb5a8e3b)

**Comment by caitlinrmcintyre** [comment]
Verified fix in IDE v2024.200.0.461  Runtime v2024.200.0.477, thanks.

**Comment by BackYard321** [comment]
> Verified fix in IDE v2024.200.0.461 Runtime v2024.200.0.477, thanks.

thank YOU!
